modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -0,0 +1,54 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.seqexec.engine.{Action, Result, Sequence, Step}
+import edu.gemini.seqexec.server.SeqexecFailure.{Execution, UnrecognizedInstrument}
+import edu.gemini.spModel.config2.{Config, ConfigSequence, ItemKey}
+import edu.gemini.spModel.obscomp.InstConstants._
+import edu.gemini.spModel.seqcomp.SeqConfigNames._
+
+import scalaz.\/
+import scalaz.NonEmptyList._
+import scalaz._
+import Scalaz._
+import scalaz.syntax.TraverseOps
+
+/**
+  * Created by jluhrs on 9/14/16.
+  */
+object SeqTranslate {
+
+  implicit def toAction[A](x: SeqAction[A]): Action = x.run map {
+    case -\/(e) => Result.Error(e)
+    case \/-(r) => Result.OK(r)
+  }
```

Comment:
That was just to not rewrite the functions that produce the TCS and instruments configuration actions. They produce SeqActions instead of actions, but they can be converted. Eventually the original functions have to be modified to produce Actions.